### PR TITLE
Project ideas for RADIS

### PR DESCRIPTION
**Pull request by erwanp**
Hello ! 

3 project ideas for RADIS 

1. Reduce Memory usage in RADIS to be able to compute terabyte-scale databases   @dcmvdbekerom
2. Model atmospheres in the RADIS spectral code with a simplified frontend and more line databases  @ulyssed
3. Add non-air diluent and custom collisional broadening in RADIS spectra  @minouHub

One last to come !

### CODE REVIEW

File: _projects/2021/radis/atmosphere_interface.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ - Git
+collaborating_projects:
+# suborganisation(s) to which this project belongs.
+ - radis
+---
+
+
+#### Description
+
+The RADIS code was developed to characterize the radiation of plasmas, flames and atmospheres. It calculate high-temperature infrared spectra from databases of tens of millions of lines, with a new algorithm that already makes it one of the fastest line-by-line spectral codes available. This performance makes it very suitable to compute high-temperature atmospheres.
```
Comment (mirca) [inline _projects/2021/radis/atmosphere_interface.md:32]: It calculate ---> It calculates
codes ---> code

File: _projects/2021/radis/non_air_diluent.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+ - Git
+collaborating_projects:
+# suborganisation(s) to which this project belongs.
+ - radis
+---
+
+
+#### Description
+
+The RADIS code was developed to characterize the radiation of plasmas, flames and atmospheres. It calculate high-temperature infrared spectra from databases of tens of millions of lines, with a new algorithm that makes it one of the fastest line-by-line spectral codes available.
```
Comment (mirca) [inline _projects/2021/radis/non_air_diluent.md:31]: see https://github.com/OpenAstronomy/openastronomy.github.io/pull/282/files#r594101025

File: _projects/2021/radis/reduce_memory_usage.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ - Git
+collaborating_projects:
+# suborganisation(s) to which this project belongs.
+ - radis
+---
+
+
+#### Description
+
+The RADIS code was developed to characterize the radiation of plasmas, flames and atmospheres. It calculate high-temperature infrared spectra from databases of tens of millions of lines, with a new algorithm that already makes it one of the fastest line-by-line spectral codes available.
```
Comment (mirca) [inline _projects/2021/radis/reduce_memory_usage.md:31]: see https://github.com/OpenAstronomy/openastronomy.github.io/pull/282/files#r594101025

**Comment by mirca** [comment]
hi @erwanp! this lgtm minor some tiny typos :) is there one more project idea to come? if so, do you want to add it to this PR or shall we merge it first so that students are aware of RADIS' projects asap? let us know :)

**Review by mirca — APPROVED** [review]

**Comment by erwanp** [comment]
Fixed, thank you ! (I kept "one of the code**s**", actually)

Let's merge now :  the last project is still being discussed.

**Comment by bsipocz** [comment]
This year there are some changes in gsoc, e.g there is only one evaluation in the middle rather than two. Could you please revise the timeline accordingly? (I think we can still merger this first to get the project on the list asap)

**Comment by erwanp** [comment]
Updated with one intermediary evaluation only !

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thanks! Any adjustments, if needed, can come in follow-up PRs.

**Pull request merged**